.github/workflows/tests.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v2
       with:
-        go-version: 1.16.3
+        go-version: 1.16.4
```

Comment:
We might want to consider just using `^1.16` in the future, to not needing to bump on each patch version, what do you think?

Reply:
+1